velox/exec/LocalPlanner.cpp

```diff
@@ -324,7 +324,7 @@ std::shared_ptr<Driver> DriverFactory::createDriver(
         auto exchangeNode =
             std::dynamic_pointer_cast<const core::ExchangeNode>(planNode)) {
       operators.push_back(std::make_unique<Exchange>(
-          id, ctx.get(), exchangeNode, exchangeClient));
```

Comment:
This change assumes that exchangeClient can be used only by one operator in the pipeline. This is probably the case, but is not intuitive. Maybe add a comment and/or a check that exchangeClient is not null.